src/Native/System.Security.Cryptography.Native/pal_err.cpp

```diff
@@ -6,15 +6,28 @@
 
 #include <openssl/err.h>
 
-extern "C" uint64_t ErrGetError(int32_t* isAllocFailure)
+extern "C" uint64_t ErrGetError()
+{
+    return ERR_get_error();
+}
+
+extern "C" uint64_t ErrGetErrorAlloc(int32_t* isAllocFailure)
 {
     unsigned long err = ERR_get_error();
 
-    *isAllocFailure = ERR_GET_REASON(err) == ERR_R_MALLOC_FAILURE;
+    if (isAllocFailure)
+    {
+        *isAllocFailure = ERR_GET_REASON(err) == ERR_R_MALLOC_FAILURE;
```

Comment:
Yes, the MALLOC_FAILURE is the only error code we needed to know about. When we need to know about more, we can make a PAL enum for it.